src/gui/properties/peerlistwidget.cpp

```diff
@@ -57,6 +57,25 @@
 #include "propertieswidget.h"
 #include "uithememanager.h"
 
+namespace BitTorrent
```

Comment:
Is it really needed to declare the following under BitTorrent namespace? One can think that it is provided by qBittorrent base...

Reply:
I'll remove it.

Reply:
Unless you meant to use anonymous namespace or something else?